include/pybind11/pybind11.h

```diff
@@ -1047,6 +1047,13 @@ inline void call_operator_delete(void *p, size_t s, size_t a) {
     #endif
 }
 
+inline void add_class_method(object& cls, const char *name_, const cpp_function &cf) {
```

Comment:
Just curious: If this PR does reduce the binary size, this `inline` is just ignored by the compiler and is only necessary to avoid the ODR?

Reply:
Correct.